### PR TITLE
Add a teeny bit more about how to use the dev cli 🙆‍♀️

### PR DESCRIPTION
**Pull request by bobcatfish**
# Changes

I want to start using the cli a bit more. I built it once but want to
keep updating it and using the bleeding edge. Also noticed there was
some copy pasta left over from the Pipeline DEVELOPMENT guide.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [n/a] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -91,7 +90,13 @@ While iterating on the project, you may need to:
 $ go build ./cmd/...
 ```
 
-It builds `tkn` binary in your current directory. You can start playing with it.
+It builds `tkn` binary in your current directory. You can start playing with
+it. If you add the directory to your `$PATH` you can use your locally built
+binary from anywhere, e.g. you can add to your `.basrc` (or similar):
+
+```
+export PATH=$PATH:$GOPATH/src/github.com/tektoncd/cli/
```
Comment (bobcatfish) [inline DEVELOPMENT.md]: @vdemeester is this what ppl working on the project do? (I'm just guessing and interested in how other folks use a dev binary and keep it up to date!)

Comment (sthaha) [reply DEVELOPMENT.md]: I have my path set to `export PATH=~/bin:$PATH` and in `~/bin/` soft link to `~/go/src/path/to/tkn` binary. I know I could use `go install` but my method keeps `tkn` upto date all the time as there is a chance I miss `go install`

Comment (bobcatfish) [reply DEVELOPMENT.md]: ah kk that makes a lot of sense! im gonna tweak this a bit then

Comment (bobcatfish) [reply DEVELOPMENT.md]: I changed this a bit @sthaha based on what you described! What do you think?

Comment (sthaha) [reply DEVELOPMENT.md]: Thanks! that's indeed a lot nicer!

**Pull request merged**